### ISSUE TITLE
Local renderer: Themes are missing backgrounds

### ISSUE THREAD

**Issue by BradyBrenot** [qjebbs/vscode-plantuml#458]
Whether I use the Preview or I export a diagram as png or svg, it seems that themes don't have the correct appearance.

In VS Code, using a local renderer:
![image](https://user-images.githubusercontent.com/632920/143086190-0de2558d-76ee-4b83-a5c4-f955de6b51e3.png)

vs. the PlantUML server:
![image](https://user-images.githubusercontent.com/632920/143086271-21dd26ab-ce61-4c61-af1b-4230797b9582.png)
and in VS Code
![image](https://user-images.githubusercontent.com/632920/143086456-4aff3aab-0492-4647-92ff-0ca0b9a75956.png)

**Comment by BradyBrenot** [comment]
This has to be something upstream, it happens when using the .jar directly as well
![image](https://user-images.githubusercontent.com/632920/143087807-837714bf-cc8d-4235-9ff1-3c48fb8aa8b6.png)

**Comment by The-Lum** [comment]
Hello,

- Could you compare the PlantUML version?
With:
```puml
@startuml
version
@enduml
```
_Because the PlantUML server is updated more often than the VS extension._
Regards.

**Comment by BradyBrenot** [comment]
Yeah, @The-Lum it looks like this is an issue specific to v1.2021.14. I put in an issue on PlantUML's repo.

https://github.com/plantuml/plantuml/issues/798

[v1.2021.13 seems fine ](https://github.com/plantuml/plantuml/releases/tag/v1.2021.13)

**Comment by BradyBrenot** [comment]
(the server is ahead of the current release)
![image](https://user-images.githubusercontent.com/632920/143090730-7f8c5455-02ac-4b85-a015-e488a4b3892d.png)

**Comment by The-Lum** [comment]
Yes, ... there are a lot of modifications/corrections about theme on the last beta... _(still under construction)_
